azure/resource_groups/ProjectCore/template.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceNamePrefix": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]"
+    }
+  },
+  "variables": {
+    "vNetName": "[concat(parameters('resourceNamePrefix'), '-vn')]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Network/virtualNetworks",
```

Comment:
Can we use the building block here?

https://github.com/DFE-Digital/bat-platform-building-blocks/blob/master/templates/virtual-network.json

It's already being used for ACI integration elsewhere.

Reply:
Oh, nice call. I assumed there wouldn't be one, yet. Thanks.